server/src/data/models.rs

```diff
@@ -4062,6 +4062,25 @@ impl From<SearchQueryEventClickhouse> for SearchQueryEvent {
     }
 }
 
+pub fn escape_quotes(value: &mut Value) {
+    match value {
+        Value::String(s) => {
+            *s = s.replace('"', "\\\"");
+        }
+        Value::Array(arr) => {
+            for item in arr {
+                escape_quotes(item);
+            }
+        }
+        Value::Object(obj) => {
+            for (_, v) in obj {
+                escape_quotes(v);
+            }
+        }
+        _ => {}
+    }
+}
+
```

Comment:
Escape Quotes section. This should fix issues around clickhouse erroring on inserts